python-3.10.yaml

```diff
@@ -0,0 +1,108 @@
+package:
+  name: python-3.10
+  version: 3.10.9
+  epoch: 0
+  description: "the Python programming language"
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: PSF-2.0
+  dependencies:
+    runtime:
+
+secfixes:
+  3.10.9-r0:
+    - CVE-2020-10735
+
+environment:
+  contents:
+    repositories:
+      - https://packages.wolfi.dev/bootstrap/stage3
```

Comment:
I would move python-3.10 down in the build order and not depend on bootstrap for it.

Reply:
This comment sounds like we shouldn't be depending on bootstrapping, am I misunderstanding the question?